### PR TITLE
[add link]

### PR DESCRIPTION
**Pull request by aangelopoulos**

### CODE REVIEW

File: backend/app/core/config.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 class Settings(BaseSettings):
     PROJECT_NAME: str = "Red-Teaming Arena"
-    ALLOWED_ORIGINS: list = ["https://www.redarena.ai", "https://redarena.ai", "http://redarena.ai"]
+    ALLOWED_ORIGINS: list = ["*"]#["https://www.redarena.ai", "https://redarena.ai", "http://redarena.ai"]
```
Comment (infwinston) [inline backend/app/core/config.py:8]: roll back?

**Review by infwinston — CHANGES_REQUESTED** [review]
@aangelopoulos thanks LGTM except for config change

**Comment by aangelopoulos** [comment]
Will do... actually will quickly try to automate so we don't have to worry about this anymore.

**Comment by infwinston** [comment]
maybe we can enable some local dev mode which will automatically assign `ALLOWED_ORIGINS=["*"]`

**Comment by aangelopoulos** [comment]
It will be easy with the .env file :)

**Review by infwinston — APPROVED** [review]

**Pull request merged**